app/models/concerns/media.rb

```diff
@@ -45,6 +47,17 @@ module Media
                          inverse_of: :item
     delegate :year, to: :start_date, allow_nil: true
 
+    scope :past, -> { where('start_date <= ?', Date.today) }
+    scope :finished, -> { past.where('end_date < ?', Date.today) }
+    scope :current, -> do
+      past.where('end_date >= ? OR end_date IS ?', Date.today, nil)
+    end
+    scope :future, -> { where('start_date > ?', Date.today) }
+    scope :upcoming, -> do
+      future.where('start_date <= ?', Date.today + 3.months)
+    end
+    scope :nya, -> { future.where('start_date > ?', Date.today + 3.months) }
```

Comment:
Could maybe change this to include all future for queries, but I did it like this to keep consistency with the attribute.